force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -355,15 +410,19 @@ public with sharing class ClassesObjectsMethods {
      * @return A list of new Person objects.
      */
     public static List<Person> createMultiplePersons(List<String> names, Integer age) {
-
+            List <Person> personObjects = new List<Person>();
         // Loop through each name in the provided list of names
             // Create a new Person object for each name
-            
+            for (Integer i = 0; i < names.size(); i++) {
+                
+                Person newPerson = new Person(names.get(i), age);
+                personObjects.add(newPerson);
+            }
             // Add the new Person object to the list of Person objects
         
         
         // Return the list of Person objects
-        return null; // Replace null with the variable you used to store the result
```

Comment:
good, if you want you can simply add to the list directly and not create a variable you don't really need. 

```
 public static List<Person> createMultiplePersons(List<String> names, Integer age) {
        List<Person> persons = new List<Person>();
        for (String name : names) {
            persons.add(new Person(name, age));
        }
        return persons;
    }
```